pom.xml

```diff
@@ -1999,7 +2000,7 @@
               <configuration>
                 <targetVersion>${maven.compiler.target}</targetVersion>
                 <!-- allow undocumented classes like sun.misc.Unsafe: -->
-                <internalRuntimeForbidden>true</internalRuntimeForbidden>
+                <internalRuntimeForbidden>false</internalRuntimeForbidden>
```

Comment:
Is this change intentional? I guess since we make the setting match the non-test code check. Just want to confirm.

Reply:
yea, without the change it complains about accessing jdk.jfr on java 8, i believe it was jdk-internal api back then